pkg/shell/templates.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+var phantom_checkpoint = phantom_checkpoint || function () { };
+
+var templates = [
+    "add-machine",
+    "auth-failed",
+    "change-auth",
+    "change-port",
+    "color-picker",
+    "host-key",
+    "not-supported",
+    "sync-users"
+];
+
+var deps = templates.map(function(template) {
+    return "data!shell/templates/" + template + ".html";
+});
+deps.unshift("base1/mustache");
```

Comment:
This seems like an overly complicated way of avoiding typing a few extra characters. I mean the strings are already hard coded above.